examples/multiedgetracking/reid/worker.py

```diff
@@ -0,0 +1,185 @@
+        """
+
+        if features == None:
+            LOGGER.warning("Target has not been set!")
+            return -1
+
+        match_id, match_score = match_query_to_targets(features, candidate_feats, False)
+        return match_id, match_score
+
+
+    def predict(self, data, **kwargs):
+        """Implements the on-the-fly ReID where detections per frame are matched with the candidate boxes."""
+        tresult = []
+
+        for dettrack_obj in data:
+            try:
+                reid_result = getattr(self, self.op_mode.value + "_no_gallery")(dettrack_obj)
+                if reid_result is not None:
+                    tresult.append(reid_result)
+                    self.store_result(reid_result)
+            except AttributeError as ex:
+                LOGGER.error(f"Error in dynamic function mapping. [{ex}]")
+                return tresult
+
+        return tresult
+
+    ### OP_MODE FUNCTIONS ###
+
+    def covid19_no_gallery(self, det_track):
+        return self.tracking_no_gallery(det_track)
+
+    def detection_no_gallery(self, det_track):
+        LOGGER.warning(f"This operational mode ({self.op_mode}) is not supported without gallery.")
+        return None
+
+    def tracking_no_gallery(self, det_track : DetTrackResult):
+        det_track.targetID = [-1] * len(det_track.bbox_coord) 
+        
+        for target in self.targets_list:
+            # get id of highest match for each userid
+            match_id, match_score = self.reid_per_frame(target.features, det_track.features)
+            
+            result = {
+                "userID": str(target.userid),
+                "match_id": str(match_score),
+                "detection_area": det_track.camera,
+                "detection_time": det_track.detection_time
+            }
+
+            if float(match_score) >= self.threshold:
+                det_track.targetID[match_id]= str(target.userid)
+                det_track.userID = target.userid
+                det_track.is_target = match_id
+
+            LOGGER.info(result)
+        
+        if det_track.targetID.count(-1) == len(det_track.targetID):
+            # No target found, we don't send any result back
+            return None
+
+        return det_track
+
+    def store_result(self, det_track : DetTrackResult):
+        try:
+            filename = save_image(det_track, self.CT, folder=f"{self.results_base_folder}{det_track.userID}/")
+            if self.enable_obs:
+                self.obs_client.upload_file(f"{self.results_base_folder}{det_track.userID}/", filename, f"/media/reid/{det_track.userID}")
+        except Exception as ex:
+            LOGGER.error(f"Unable to save image: {ex}") 
+
+class Bootstrapper(Thread):
+    def __init__(self):
+        super().__init__()
+
+        self.daemon = True
+        self.folder = "/data/"
+        self.retry = 3
+        self.job = ReID(models=[ReIDWorker()], asynchronous=False)
```

Comment:
Interface unification between Multi-Edge Tracking with other feature interface needs to be considered. At least in other feature, the parameter is called `estimator`, instead of `models`